source/includes_main/_prospects.md

```diff
@@ -110,6 +105,12 @@ list_name | no | **string** <br />The name of the list if the prospect is in a l
 created_at | no | **datetime** | ISO 8601 format with timezone offset
 updated_at | no | **datetime** | ISO 8601 format with timezone offset
 
+### Custom Fields
+
+Custom fields allow you to store more specific information about your prospects, they can be defined into [your settings](https://app.prospect.io/settings/custom-fields).
+
+Use the `identifier` of your custom fields as attribute keys.
```

Comment:
Is it good to introduce what do they are here? 

Instead, I'd suggest: 

Custom fields can be accessed as normal fields by using their "identifiers". 
Ex: c_custom_field_a 
The list of custom fields is available under the endpoint #... link to endpoint

Reply:
Good point, I've changed ;)